webhook.py

```diff
@@ -39,14 +39,17 @@ async def webhook_if_message_altered(original: discord.Message, copy: MessageCop
     This function calls the proxy_message_by_webhook function if the message copy
     has been altered in any way by the on_message event listeners in main.py
     '''
-    if original.content != copy.content or original.author.display_name != copy.display_name or original.author.avatar_url != copy.avatar_url:
+    if original.content != copy.content or original.author.display_name != copy.display_name or original.author.avatar_url != copy.avatar_url or original.attachments != copy.attachments:
```

Comment:
Fine for now, though we may want to clean this up to use `if any()` instead?

Reply:
I cleaned it a bit. Better now than when it actually gets too hard to read